Jasmine/Common/ViewModels/LevelImportViewModel.swift

```diff
@@ -0,0 +1,70 @@
+import RealmSwift
+
+class LevelImportViewModel {
```

Comment:
Should conform to LevelImportViewModelProtocol in my `level-selector-vc` branch.